idearium-lib/lib/mq/manager.js

```diff
@@ -44,26 +46,30 @@ class Manager extends EventEmitter {
      * Return an instance of the Loader class, configured as required.
      * @return {Loader} An instance of the Loader class.
      */
+    // eslint-disable-next-line no-restricted-syntax
     get loader () {
 
-        if (!this._loader) {
-            this._loader = new Loader();
-            this._loader.camelCase = false;
+        if (!this.loaderInstance) {
+
+            this.loaderInstance = new Loader();
+            this.loaderInstance.camelCase = false;
+
         }
 
-        return this._loader;
+        return this.loaderInstance;
 
     }
 
     /**
      * Register message queue consumers
      * @return {[type]} [description]
      */
-    registerConsumers() {
+    registerConsumers () {
 
-        var messages = this.messages;
+        const { messages } = this;
 
         return Promise.all(Object.keys(messages)
+            // eslint-disable-next-line no-undefined
             .filter(messageName => messages[messageName].consume !== undefined)
```

Comment:
I think this should be updated so we follow the eslint rule.
`typeof messages[messageName].consume !== 'undefined'`

Reply:
Fixed :)